app/react/Entities/EntitiesAPI.js

```diff
@@ -1,10 +1,15 @@
-/** @format */
-
 import api from 'app/utils/api';
+import { RequestParams } from 'app/utils/RequestParams';
 
 export default {
-  get(requestParams) {
-    return api.get('entities', requestParams).then(response => response.json.rows);
+  get(requestParams = new RequestParams()) {
+    const params = requestParams.add({
+      include:
+        requestParams.data && requestParams.data.include
+          ? requestParams.data.include.concat(['permissions'])
+          : ['permissions'],
```

Comment:
@daneryl, more as a question than a change request, is not this the same that keeping burned "+permissions" into the route action? I was thinking the intention of adding the include parameter is to avoid getting the permissions property in several places at the frontend.

Reply:
@mfacar the intention was for the api not to return permissions by default, in our own stack looks the same, but now you have you explicitly ask for the permissions, does this answers the question ?

Reply:
Totally, thank you, I'll proceed with the QA